src/components/MoneyReportHeader.tsx

```diff
@@ -241,6 +250,20 @@ function MoneyReportHeader({
                     </View>
                 )}
             </HeaderWithBackButton>
+            {allHavePendingRTERViolation && (
+                <MoneyRequestHeaderStatusBar
+                    title={
+                        <Icon
+                            src={Expensicons.Hourglass}
+                            height={variables.iconSizeSmall}
+                            width={variables.iconSizeSmall}
+                            fill={theme.textSupporting}
```

Comment:
actually I think it should. Based on that https://github.com/Expensify/App/issues/39533#issuecomment-2079398108 

I have changed it here and the other places to `theme.icon`. I was misled because `theme.textSupporting` wa used for icons of different violations somewhere else in the code.